models/cartDao.js

```diff
@@ -0,0 +1,146 @@
+const { appDataSource } = require("./appDataSource");
+
+const getItems = async (userId) => {
+  try {
+    const cartData = await appDataSource.query(
+      `SELECT 
+        c.user_id AS userId,
+        c.id AS cartId,
+        c.user_id AS userId,
+        p.id AS productId,
+        c.product_option_id AS productOptionId,
+        p.name AS productName,
+        p.main_image AS productMainImage,
+        p.price AS productPrice,
+        p.discount_price AS productDiscountPrice,
+        po.name AS productOptionName,
+        po.extra_price AS productOptionExtraPrice,
+        p.discount_price + po.extra_price AS productTotalPrice,
+        po.inventory AS inventory,
+        c.quantity AS quantity,
+        ((p.discount_price + po.extra_price ) * c.quantity) AS productTotalPriceWithQuantity
+      FROM carts c
+      JOIN product_options po ON c.product_option_id = po.id
+      JOIN products p ON po.product_id = p.id
+      WHERE user_id = ?
+      ORDER BY p.id, po.id;
+      `,
+      [userId]
+    );
+    return cartData;
+  } catch (err) {
+    err.message = "FAIL TO GET ITEMS IN CART";
+    err.statuscode = 500;
+    throw err;
+  }
+};
+
+const addItem = async (userId, productOptionId, quantity) => {
+  try {
+    return await appDataSource.query(
```

Comment:
```suggestion
    return appDataSource.query(
```
이후로 동기적인 작업을 기다리는 코드가 없기 때문에 await는 지워주셔도 될 것 같습니다.